### ISSUE TITLE
MKL 2024.0 removed support for osx-64

### ISSUE THREAD

**Issue by h-vetinari** [conda-forge/intel_repack-feedstock#62]
#61 intentionally removed support for osx-64. I'm guessing that's permanent and not coming back?

[This is intended as more of an announcement; it's not a complaint]

_Originally posted by @h-vetinari in https://github.com/conda-forge/intel_repack-feedstock/issues/61#issuecomment-1886246931_

**Comment by milubin** [comment]
yes, as @napetrov said in #61 there are no MacOS packages created by Intel starting from 2024.0 release.
